src/apps/reservation-list/reservation-material/reservation-info.tsx

```diff
@@ -46,6 +44,23 @@ const ReservationInfo: FC<ReservationInfoProps> = ({
         });
   }
 
+  const config = useConfig();
+  // Get library branches from config
+  const inputBranches = config<AgencyBranch[]>("branchesConfig", {
+    transformer: "jsonParse"
+  });
+
+  // Get the library branches where the user cannot pick up books at
+  const blacklistBranches = config("blacklistedPickupBranchesConfig", {
+    transformer: "stringToArray"
+  });
+
+  // Remove the branches where the user cannot pick up books from the library branches
+  let branches = inputBranches;
+  if (Array.isArray(blacklistBranches)) {
+    branches = excludeBlacklistedBranches(inputBranches, blacklistBranches);
+  }
```

Comment:
Kasper B. has made a hook that does something similar, so if you can use it I think we should not repeat the functionality. Please take a look at it:
https://github.com/reload/dpl-react/blob/cf64e43d25211f3a141e6ea195f031dc1c88d654/src/core/utils/branches.tsx#L26

Reply:
Jep ill change

Reply:
This doesn't work with a function I use below, https://github.com/danskernesdigitalebibliotek/dpl-react/blob/c61d078d0a51f5bc78991b929c28fdd4ef6869b5/src/components/reservation/helper.ts#L23 i'll add a todo for now

Reply:
Too bad that the hook does not work in your context. 
I think the correct approach would be to make the hook work in both cases - with a memo hook so we only need to make the branch filtering once.
But I respect that there is a time schedule to follow. :)

BUT! it looks like the whole code block is being repeated, so please make a function for it.